Formula/python-launcher.rb

```diff
@@ -0,0 +1,31 @@
+class PythonLauncher < Formula
+  desc "Launch your Python interpreter the lazy/smart way"
+  homepage "https://github.com/brettcannon/python-launcher"
+  url "https://github.com/brettcannon/python-launcher/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "31e5a4e50e3db4506e8484db06f6503df1225f482b40a892ffb5131b4ec11a43"
+  license "MIT"
+
+  head "https://github.com/brettcannon/python-launcher.git", branch: "main"
+
+  livecheck do
+    url :head
+    regex(/^v(\d+\.\d+\.\d+)$/i)
+  end
```

Comment:
Do we need this?

Reply:
We can remove this, there aren't any non-standard tag names in the repository.